src/connectors/index.js

```diff
@@ -4,10 +4,8 @@ import admin from './admin.js'
 export default function (){
 
 const apiUrl = "http://localhost:10000"
-const accountsConnectors = accounts(fetch, apiUrl)
+const accountsConnectors = accounts(fetch, "http://localhost:20000")
```

Comment:
We should use the config files for this. Since this frontend will be used in many project, we should probably include the config via a script tag in the main html file.